new.txt

```diff
@@ -0,0 +1,5 @@
+1.�� �� ������� HTML �������� ��������� ������� ���:
```

Comment:
cp1251?

Reply:
![image](https://user-images.githubusercontent.com/209223/77540948-a92dc200-6eac-11ea-85d8-3425b1db02dd.png)

Reply:
The text was in Cyrillic and therefore had a problem.